packages/blade/src/components/Typography/BaseText/BaseText.native.tsx

```diff
@@ -70,7 +68,7 @@ export const BaseText = ({
       fontStyle={fontStyle}
       textDecorationLine={textDecorationLine}
       lineHeight={lineHeight}
-      as={as}
+      as={undefined}
```

Comment:
Note: in react native i'm not even passing down the as prop since anyways we aren't throwing the error above.